README-PNL.md

```diff
@@ -86,15 +113,16 @@ Launch a DPdash instance as follows:
     
     singularity run -B ${state}:/data -B ${data}:/project_data ${DPDASH_IMG} /sw/apps/dpdash/singularity/run.sh
     
-Alternatively, you can shell into the container and then execute further commands. In fact, this would be 
-the recommended method for working with the image.
+Alternatively, you can shell into the container and then execute further commands. In fact, this would be the recommended method for working with the image.
     
     singularity shell -B ${state}:/data -B ${data}:/project_data ${DPDASH_IMG}
-    
+
     Singularity> cd /sw/apps/dpdash/singularity/
     Singularity> ./run.sh
 
-At these point, you can even exit from the Singularity shell and yet your DPdash instance would be running.
+At this point, you can even exit from the Singularity shell and yet your DPdash instance would be running.
+
+**Note:** We also provide a convenience script in the `singularity` directory called `start.sh`, which can be run on its own to launch the DPDash instance, or run with the `-s` flag to shell into the Singularity container.
```

Comment:
Can we append something like:

> However, `start.sh` (and `stop.sh`) scripts come with a fixed `singularity` command. If you need to mount arbitrary directories or modify the `singularity` command, you will not be able to use the convenience scripts. In that case, use `singularity` command directly in your terminal as shown above.

Reply:
Good idea!

Reply:
Done in ff44a9f86795c5848c887f4e3a83a53a3604e966.